### ISSUE TITLE
Jogl 2.4 and Java 8 FloatBuffer NoSuchMethodError

### ISSUE THREAD

**Issue by ncollier** [Repast/repast.simphony#33]
Jogl 2.4 and Java 8:

```
caused by: java.lang.NoSuchMethodError: java.nio.FloatBuffer.rewind()Ljava/nio/FloatBuffer;
    at saf.v3d.grid.GridMesh.updateColors(GridMesh.java:119)
    at saf.v3d.grid.GridMesh.update(GridMesh.java:115)
    at saf.v3d.grid.VGridShape.update(VGridShape.java:25)
    at repast.simphony.visualizationOGL2D.ValueLayerDisplayLayer.update(ValueLayerDisplayLayer.java:67)
    at repast.simphony.visualizationOGL2D.DisplayOGL2D.update(DisplayOGL2D.java:402)
    at repast.simphony.visualizationOGL2D.DisplayOGL2D.reshape(DisplayOGL2D.java:589)
    at saf.v3d.Canvas2D.reshape(Canvas2D.java:400)
    at jogamp.opengl.GLDrawableHelper.reshape(GLDrawableHelper.java:751)
    at com.jogamp.opengl.awt.GLJPanel$Updater.display(GLJPanel.java:1442)
    at com.jogamp.opengl.awt.GLJPanel$10.run(GLJPanel.java:1521)
    at jogamp.opengl.GLDrawableHelper.invokeGLImpl(GLDrawableHelper.java:1293)
```

**Comment by ncollier** [comment]
Possible solution from etatara:

https://github.com/plasma-umass/doppio/issues/497

**Comment by ncollier** [comment]
Potential fix in b044a60. Need etatara to test.
